doc/installing.md

```diff
@@ -66,6 +66,49 @@ If you prefer a different installation method, see {ref}`installing`.
    This will create a minimal setup with default options.
    If you want to tune the initialization options, see {ref}`initialize` for more information.
 
+#### Gentoo
+Incus and all of its dependencies are available in Gentoo's main repository as [`app-containers/incus`](https://packages.gentoo.org/packages/app-containers/incus).
+
+Install Incus with:
+
+    emerge -av app-containers/incus
+
+Note: Installing LTS vs. feature-release will be explained later, when Incus upstream and Gentoo's repository has those releases available.
+
+There will be two newly created groups associated to Incus: 
+`incus` to allow basic user access (launch containers), and `incus-admin` for `incus admin` controls. Add your regular users to either, or both, depending on your setup and use cases.
+
+After installation, you may want to configure Incus. This is optional though, as the defaults should also just work.
+
+ - **openrc**: edit /etc/conf.d/incus
+
+ - **systemd**: systemctl edit --full lxd.service
+
+Set up subuid and subgid:
+
+    echo "root:1000000:1000000000" | tee -a /etc/subuid /etc/subgid
+
+For more information: [`Gentoo Wiki / subuid/subgid`](https://wiki.gentoo.org/wiki/Subuid_subgid)
```

Comment:
Wasn't sure if this or https://github.com/lxc/incus/blob/main/doc/userns-idmap.md was better to link here.

Reply:
I think so as most distributions will ahve the same "problem" when uidmap is in place on the system, so probably best to send everyone to the same package for consistency.

```
{ref}`{userns-idmap}`
```
should do the trick

Reply:
Couldn't get that, or ``{ref}`userns-idmap`` ` to work. But this current way seems to.